test-it/karma/utilities/TestHelpers.VireoRunner.js

```diff
@@ -5,7 +5,7 @@
     'use strict';
     window.testHelpers = window.testHelpers || {};
 
-    var rebootAndLoadVia = function (vireo, viaAbsolutePath) {
+    var rebootAndLoadVia = function (vireo, viaAbsolutePath, isDebuggingDisabled) {
```

Comment:
Based on the other feedback we should change this name to config and pass it through to loadVia

Reply:
Done.